docs/vsphere_ci/README.md

```diff
@@ -16,169 +21,113 @@ In case of our vSphere environment, we need to install Packer 1.6.6 on the basti
-         "guest_os_type":"windows9Server64Guest",
-         "insecure_connection":"true",
-         "iso_paths":[
-            "{{user `os_iso_path`}}",
-            "<tools_iso_path>"
-         ],
-         "network_adapters":[
-            {
-               "network":"{{user `vsphere-network`}}",
-               "network_card":"vmxnet3"
-            }
-         ],
-         "password":"{{user `vsphere-password`}}",
-         "storage":[
-            {
-               "disk_size":"{{user `vm-disk-size`}}",
-               "disk_thin_provisioned":true
-            }
-         ],
-         "type":"vsphere-iso",
-         "username":"{{user `vsphere-user`}}",
-         "vcenter_server":"{{user `vsphere-server`}}",
-         "vm_name":"{{user `vm-name`}}",
-         "ssh_password":"{{user `winadmin-password`}}",
-         "ssh_username":"Administrator",
-         "ssh_timeout":"50m"
-      }
-   ],
-   "provisioners":[
-      {
-	"pause_before": "120s",
-	"inline":["ipconfig"],
-        "type":"powershell",
-	"elevated_user": "Administrator",
-        "elevated_password": "{{user `winadmin-password`}}",
-	"max_retries":"10"
-      }
-   ],
-   "sensitive-variables":[
-      "vsphere_password",
-      "winadmin_password"
-   ],
-   "variables":{
-      "os_iso_path":"<Windows_os_iso_path>",
-      "vm-cpu-num":"2",
-      "vm-disk-size":"128000",
-      "vm-mem-size":"4096",
-      "vm-name":"<template_to_be_created>",
-      "vsphere-cluster":"Cluster-1",
-      "vsphere-datacenter":"<datacenter_name>",
-      "vsphere-datastore":"<datastore_name>",
-      "vsphere-folder":"<folder_where_template_gets_built>",
-      "vsphere-network":"network_segment_name",
-      "vsphere-password":"<password>",
-      "vsphere-server":"<server_name>",
-      "vsphere-user":"<user_name>",
-      "winadmin-password":"<Windows_os_password>"
-   }
-}
-```
 
 ### Variables
-- `<tools_iso_path>` - Path where VMWare tools iso is available in vSphere datacenter
-- `<Windows_os_iso_path>` - Path where Windows ISO was dowloaded to in vSphere datacenter
-- `<template_to_be_created>` - Name of the Windows vSphere template that will be created by successful Packer run in 
- 			       vSphere datacenter
-- `<datacenter_name>` - Name of the vSphere datacenter
-- `<datastore_name>` - Name of the vSphere datastore
-- `<folder_where_template_gets_built>` - Path where Windows template gets built by successful Packer run in 
-					 vSphere datacenter
-- `<password>` - vSphere environment password
-- `<server_name>` - vSphere environment URL
-- `<user_name>` - vSphere username to login
-- `<Windows_os_password>` - Password for the Windows template
 
-## What actually happens during build
+In order to use the provided [reference build file](build.json) as a valid configuration with Packer, you **must** 
+adjust the following variables:
+
+- `<vmtools-iso-path>` Path where VMWare Tools ISO is available in vSphere datacenter (i.e. `[] /usr/lib/vmware/isoimages/windows.iso`)
```

Comment:
Rather than `i.e.` do you mean `example:`? Why is the `[]` needed?

Reply:
> Rather than i.e. do you mean example:? 

No, [ that is ](https://thewordcounter.com/i-e-meaning/) a valid value, not an example.

> Why is the [] needed?

In vSphere, that's the default path to the VMWare Tools ISO image for Windows. `[]` means no vSAN Datastore is specified. Otherwise, `[WorkloadDatastore] path/to/folder` would be a valid path.

Reply:
So is that a constant or can it change? `i.e.` implies it is constant for all vSphere clusters and I am not so sure if that is the case.

Reply:
That's a good question. To my understanding it is not a constant, but rather is a default in vSphere. 

The iso images directory is [a symlink to /productLocker/vmtools](https://kb.vmware.com/s/article/1037405). 

Happy to follow your lead here, just wanted to present a valid default value.

Reply:
I would just say `... in the vSphere datacenter (default: '[] /usr/lib/vmware/isoimages/windows.iso\')`